Makefile

```diff
@@ -1,13 +1,12 @@
+CPUS ?= $(shell sysctl -n hw.ncpu || echo 1)
```

Comment:
I like how this only works on OSX. I think you'll break it in Travis. Or I guess just fall back to 1.